### PR TITLE
add responsiveness to background floating items and video modal

### PR DESCRIPTION
**Pull request by rileydanejohnston**
- add media queries to the background floating items
- add small style changes to the video modal
- refactor styledLandingPage

**Note: You may notice the floating icons are not where they should be on the page when you compare with the design. I believe this is because the form section is not responsive yet. For example, the background section on the design might be 1000px, but on this PR it's only 750px because the form section isn't pushing all its content down the page. I believe that once the form section is responsive these background items will be in the correct place.

I have a table of the calculations I used to come up with the percentages for the background items. I'm happy to share it if needed. 

** also: The responsive section of the design doesn't have the video modal while open, so I used my best judgement for now. All I did was scale the video size down proportionally and move the close button on top instead of the side.

- [ ] My code is covered by unit tests 
- [ x] This PR is less than [300 lines of code](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests)
- [ x] My code [smells good](https://en.wikipedia.org/wiki/Code_smell) 
- [ x] I have tested my PR locally and proven it works. 
- [ x] There is [no copy-paste](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

### CODE REVIEW

File: frontend/src/components/landing/styledLanding.js [hunk 1]
```diff
@@ -69,15 +69,115 @@ export const FounderSmallText = styled(SmallText)`
...
+  right: -40px;
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+  transition: opacity 0.15s ease;
+  &:hover {
+    opacity: 0.7;
+  }
+
+  @media (max-width: 480px){
```
Comment (McCambley) [inline frontend/src/components/landing/styledLanding.js:133]: Looks great! I think this is what Danila would advice too 👍

File: frontend/src/components/landing/Landing.js [hunk 2]
```diff
@@ -89,7 +96,7 @@ function Landing({ setLoggedIn }) {
       </FounderContainer>
       <Cards>
         <Card1 translate>
-          <CardHeading>1. You create form</CardHeading>
+          <CardHeading $width='316px'>1. You create a form</CardHeading>
```
Comment (McCambley) [inline frontend/src/components/landing/Landing.js:99]: I don't believe that these need to be preceeded with a `$`, just so you know. Those are only for boolean props that you'd pass, like `$isOpen` or `$mobileWidth`. This works, and is fine, but it's not needed, just so you know.

File: frontend/src/components/landing/styledLanding.js [hunk 3]
```diff
@@ -206,88 +306,137 @@ export const FloatingIcon = styled.img`
...
-  min-width: 100vw;
-  min-height: 100vh;
-  z-index: 10;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  background-color: rgba(0, 0, 0, 0.5);
-  transition: visibility 0.3s ease, opacity 0.3s ease;
-`;
+export const PurpleBar = styled.img`
```
Comment (McCambley) [inline frontend/src/components/landing/styledLanding.js:309]: These all look beautiful. Everything is in place. Calculating all these percentages must have been a doozy, nice job.

**Review by McCambley — APPROVED** [review]
Nice Riley, looks good to me! Nice job with all the calculations, everything looks good. I imagine you're right, that once the form gets into place, things will be more aligned with how they are in the design. That being said, things look good as they are.

File: frontend/src/components/landing/Landing.js [hunk 4]
```diff
@@ -102,54 +109,38 @@ function Landing({ setLoggedIn }) {
...
-          alt="three purple stick"
-          width="276px"
-          top="570px"
-          left="62.5%"
-        />
-        <FloatingIcon src={eyes} alt="eyes" width="245px" top="922px" left="29%" />
-        <FloatingIcon src={bigStar} alt="big star" width="208px" top="1181px" left="70.7%" />
-        <FloatingIcon src={smallStar} alt="small star" width="138px" top="1154px" left="66.5%" />
-        <FloatingIcon src={phone} alt="phone" width="344px" top="1400px" left="18%" />
+        <PurpleBar src={purpleBar} alt='purple stick' />
```
Comment (cjmaret) [inline frontend/src/components/landing/Landing.js:135]: cool! the media queries look excellent, and this looks much cleaner

**Review by cjmaret — APPROVED** [review]

**Review by cjmaret — APPROVED** [review]

**Pull request merged**